tests/admin_views/tests.py

```diff
@@ -5432,6 +5433,19 @@ def get_preserved_filters_querystring(self):
             '_changelist_filters': self.get_changelist_filters_querystring()
         })
 
+    def get_unicode_changelist_filters(self):
```

Comment:
This is only used once, so I don't see why it needs to be a method.

Reply:
Oh sorry, I just followed the way of method `get_changelist_filters()`. It is also used once. 
So should I remove my `get_unicode_changelist_filters()` and also the `get_changelist_filters()`? Or just mine?

Reply:
Only change your own code, please. We don't like to mix unrelated cleanups with bug fixes.